index.html

```diff
@@ -71,9 +71,13 @@
 are starting to be developed and deployed.
 A large number of jurisdictions are starting
 to create digital identity systems for their citizens.
+
 The [Federated Identity Working Group](https://www.w3.org/2025/02/wg-fedid.html)
 includes an API for accessing these systems
 as a chartered deliverable.
+This API follows a three-party model
+where the primary actors are issuer, holder, and verifier;
+such as that defined in [[VC-DATA-MODEL-2.0]].
```

Comment:
I'm not sure we need this given the links, but if we keep it, I think "such as" isn't quite right. That implies that vc-data-model provides examples, but it's more that the definitions match. Perhaps:


```suggestion
matching the definitions in [[VC-DATA-MODEL-2.0]].
```

or if we want to say that vc-data-model is authoritative for this

```suggestion
as defined in [[VC-DATA-MODEL-2.0]].
```